src/encoder.rs

```diff
@@ -805,7 +805,7 @@ impl<W: io::Write> UncompressedHeader for BitWriter<W, BigEndian> {
 
           self.write_bit(true)?; // trailing bit
           self.byte_align()?;
-          return Ok((()));
+          return Ok(());
```

Comment:
```suggestion
          Ok(())
```
Maybe?

Reply:
In this case, since it's an early return, we need the `return` keyword. However, since there was only one instance of the "extra parens" lint being triggered, and it was in a return statement, it seemed reasonable to group it in with this PR, hence this line being changed.

Reply:
I see, I didn't check the whole codeflow :)